network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -436,10 +435,20 @@ class BanneredCampaignPage(PrimaryPage):
 
     show_in_menus_default = True
 
+    def get_intro(self):
+        if self.use_intro_from_parent:
+            parent = self.get_parent()
+            if parent:
```

Comment:
Can this fail? I think except for the root object all pages will have a parent.

Reply:
Oops, guess i just got so used to my real life `parents === None`  😂 

Sorry bad joke haha

Reply:
I think it's fine to be defensive about these things. Maybe we just make this more explicit by defining the return value for that add a test for the case.